Taskfile.yml

```diff
@@ -16,6 +16,34 @@ tasks:
       - sh -c "tar -xzf ./{{.FREESCOUT_VERSION}}.tar.gz --strip-components 1 -C freescout"
       - sh -c "rm {{.FREESCOUT_VERSION}}.tar.gz"
 
+  dist:update:
```

Comment:
Maybe we should add and database dump before upgrade? To be able to roll back fast?